crates/miden-objects/src/batch/proven_batch.rs

```diff
@@ -12,6 +14,8 @@ use crate::{
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct ProvenBatch {
     id: BatchId,
+    reference_block_commitment: Digest,
+    reference_block_num: BlockNumber,
```

Comment:
nit (feel free to ignore): maybe shorten `reference_` to `ref_` for succinctness?

Reply:
I'm fine with the short version generally, but "ref block" or "block ref" doesn't sound as clear to me as "reference block", so I'd leave it as-is, if that's fine.